src/push/plugin.ts

```diff
@@ -88,13 +88,14 @@ export function MultiAssetPlugin(callback: PluginCallback): esbuild.Plugin {
         const entryPath =
           path.join(path.dirname(args.importer), args.path) +
           path.extname(args.importer);
+
         // Spin off another build to copy over the imported modules without bundling
         const result = await esbuild.build({
           ...commonOptions(),
           entryPoints: {
             [entryPath]: entryPath,
           },
-          bundle: false,
+          bundle: true,
```

Comment:
this would change our journey files format IIRC.

Reply:
That's interesting, do you mean the file format itself (the extension)? Or do you mean the content format (i.e. bundling the CommonJS infra)?

In my tests, I added an `fs.writeFile` to the code so I could save the generated zips to my disk. Then, once I opened them, I did manage to see that the file extension was `.js` and that the content was also valid JavaScript.

I also checked the docs here: https://esbuild.github.io/api/#bundle and couldn't see a reference for format changes.

Can you go into a bit more detail about your concern here? Perhaps I'm missing something or misunderstood.

Reply:
I mean the content of the journey itself. Lets say when you push a journey file `file.journey.ts`, What does the content look like? Did it get changed to something else and whether the files were bundled - concatenated in to a single file for the files(utils, helpers) that were imported by the journey?